app/helpers/application_notifications_helper.rb

```diff
@@ -0,0 +1,16 @@
+module ApplicationNotificationsHelper
+  # Returns the view path of the notification
+  #
+  # @param [Course::Notification|UserNotification] notification The target notification
+  # @return [String] The view path of the notification
+  def notification_view_path(notification)
```

Comment:
Is this supposed to be used only in the view, or also in the controller? If it is supposed to be in the controller it should be a concern. If it is only the view then it's a view helper.